PittAPI/textbook.py

```diff
@@ -1,7 +1,30 @@
 import grequests
 import requests
 
+from bs4 import BeautifulSoup
+from requests.exceptions import ConnectionError as RequestsConnectionError
+from typing import List, Dict, Any, Callable, Generator, Tuple
+
 BASE_URL = 'http://pitt.verbacompare.com/'
+
```

Comment:
The `_fetch_term_codes` method was recently removed in pull request #73 .
Please remove the method, the `TERMS` constant, and the `requests.exception` import.